pds_doi_core/outputs/osti.py

```diff
@@ -21,4 +24,27 @@ def create_osti_doi_reserved_record(self, doi_record_list):
         renderer = pystache.Renderer()
         return renderer.render_path('config/DOI_IAD2_reserved_template_20200205-mustache.xml', {'dois': doi_record_list})
 
+    def create_osti_doi_release_record(self, doi_record_list):
```

Comment:
I am not fully sure if we need this function since we should be able to release a 'reserve' or 'draft' output as-is.

If you confirm that, can you remove this code ? We would not need the mustache template either.

Reply:
Can confirm that if we send the document directory from user command to OSTI, there's no need to do any parsing.  Both the function to create release record and the pystache file have removed.

Reply:
Ok cool